orchestra/utils/assignment_snapshots.py

```diff
@@ -26,7 +29,14 @@ def load_snapshots(snapshots):
             snapshot_data[TIME_WORKED] = snapshot_data.get(TIME_WORKED, 0)
         snapshots[SNAPSHOTS] = snapshot_list
         snapshots[VERSION] = 1
-    if snapshots[VERSION] != 1:
+    if snapshots[VERSION] == 1:
+        snapshot_list = snapshots.get(SNAPSHOTS, [])
+        for snapshot_data in snapshot_list:
+            snapshot_data[DATETIME] = parse(
+                snapshot_data[DATETIME]).replace(tzinfo=tzutc()).isoformat()
+        snapshots[SNAPSHOTS] = snapshot_list
+        snapshots[VERSION] = 2
```

Comment:
so the goal is to support old snapshots at varying versions? Can we at least pull this out into a `migrate_snapshot(from, to)` method? Also maybe call out the latest version somewhere?